test/test_ops.py

```diff
@@ -1676,6 +1767,7 @@ def test_ciou_loss(self, dtype, device):
         def assert_ciou_loss(box1, box2, expected_output, reduction="none"):
 
             output = ops.complete_box_iou_loss(box1, box2, reduction=reduction)
+            # TODO: When passing the dtype, the torch.half test doesn't pass...
```

Comment:
Is it still valid?

Reply:
Can we provide a bit more info on what doesn't pass here and what's exactly the issue?

Reply:
@datumbox I read the cIoU code since it was passing the torch.half tests and I found out that the dtype wasn't passed so the test wasn't correct for torch.half. For now, I have removed the dtype to have the same code as cIoU but I think we should investigate this further (or maybe we can't do anything since we use the _upcast function? 🤔). Let me know if this clear enough. I can provide more details.